core/txpool/legacypool/legacypool.go

```diff
@@ -546,15 +546,19 @@ func (pool *LegacyPool) Pending(filter txpool.PendingFilter) map[common.Address]
 
 	// Convert the new uint256.Int types to the old big.Int ones used by the legacy pool
 	var (
-		minTipBig  *big.Int
-		baseFeeBig *big.Int
+		minTipBig       *big.Int
+		baseFeeBig      *big.Int
+		maxDASizeScaled *big.Int
```

Comment:
It's not immediately obvious that this is in units of MB.

Reply:
Yes this is rather ugly. The context:  all the l1 calculations are "scaled" by 1e6 so that we can have sub-integer precision using integer artithmetic.  We implemented the regression to assume these scaled inputs.  Using "Scaled" in the variable/function name is the current idiom used to indicate this.

Reply:
We could also take a different approach where we just divide the returned value from `types.EstimatedL1SizeScaled` by 1e6. This way, we don't need to handle scaled values here. If I see correctly, we don't use the scaling for more precise calculations in the context here anyways.

Reply:
yeah I like that, done.